Formula/bit.rb

```diff
@@ -2,15 +2,8 @@
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "https://bitsrc.jfrog.io/bitsrc/bit-brew/stable/bit/0.1.12/bit-0.1.12-brew.tar.gz"
-  sha256 "122ec737cca207bba1fa9ae6e3ea22acf53cecd56b542a2a5bacd380a88bdef9"
-
-  bottle do
```

Comment:
Yes but you are removing `bottle` block, this is replace by test bot automatically.